<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar Pinturas</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Buscar Pinturas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>468</width>
        <height>540</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBoxTitulo">
         <property name="title">
          <string>Título</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <layout class="QVBoxLayout" name="tituloLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLineEdit" name="titulo_busca">
            <property name="placeholderText">
             <string>Digite o título ou parte do título</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_intervalo_titulo">
            <property name="text">
             <string>Busca por intervalo de letras</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frameIntervalo">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="intervaloLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QLineEdit" name="letra1">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>De</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sinal1">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>&gt;=</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="letra2">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Até</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sinal2">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>&lt;=</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxData">
         <property name="title">
          <string>Data</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <layout class="QVBoxLayout" name="dataLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QFrame" name="frameDataSimples">
            <layout class="QHBoxLayout" name="dataSimples">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QDateEdit" name="data_ini">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_data_op">
               <item>
                <property name="text">
                 <string>=</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&gt;=</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;=</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="dataSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_data_periodo">
            <property name="text">
             <string>Buscar por período</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="framePeriodo">
            <layout class="QHBoxLayout" name="periodoLayout">
             <property name="spacing">
              <number>8</number>
             </property>
             <item>
              <widget class="QLabel" name="label_ate">
               <property name="text">
                <string>até:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="data_fin">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="periodoSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxDetalhes">
         <property name="title">
          <string>Detalhes</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <layout class="QFormLayout" name="detalhesLayout">
          <property name="horizontalSpacing">
           <number>8</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelTecnica">
            <property name="text">
             <string>Técnica:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="tecnica_busca">
            <property name="placeholderText">
             <string>Ex: Óleo sobre tela</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSerie">
            <property name="text">
             <string>Série:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_serie"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelLocal">
            <property name="text">
             <string>Local:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="local_busca">
            <property name="placeholderText">
             <string>Local atual da obra</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTamanho">
         <property name="title">
          <string>Tamanho</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <layout class="QHBoxLayout" name="tamanhoLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelLargura">
            <property name="text">
             <string>Largura:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tamanho_x">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAltura">
            <property name="text">
             <string>Altura:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tamanho_y">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="tamanhoSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttonFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <spacer name="buttonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirma_busca">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelar_busca">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_intervalo_titulo</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameIntervalo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_data_periodo</sender>
   <signal>toggled(bool)</signal>
   <receiver>data_fin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelar_busca</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
